pkg/testrunner/template/render.go

```diff
@@ -52,6 +51,7 @@ func renderSingleChart(renderer chartrenderer.ChartRenderer, parameters *Testrun
 			"autoscalerMax":        parameters.AutoscalerMax,
 			"floatingPoolName":     parameters.FloatingPoolName,
 			"loadbalancerProvider": parameters.LoadBalancerProvider,
+			"gardenerVersion":      parameters.GardenerVersion,
```

Comment:
I would create a new object in the helm vallues file for the gardener version.
So that it will be at `.Values.gardener.version` instead of being in the shoot values `.Values.shoot.gardenerVersion`

Reply:
done